oak_loader/.dockerignore

```diff
@@ -2,4 +2,4 @@
 **
 
 # Allow only the binaries we care about
-!/target/x86_64-unknown-linux-musl/release/**
+!/bin/**
```

Comment:
Out of curiousity: I am not sure what we allow here, but I am a bit confused by the path. I assume we want the to allow `workspace/bin/**` here because nearly every binary is copied there now. But what was allowed before---did we have a `workspace/target/...` folder?

Reply:
Previously, this used to be invoked from the context of this package, and target referred to the local target folder. Now, it is invoked at the workspace root, and there we also have the bin folder, so I decided to use that. If we remove --out-dir, we may have to change this to target again, though still relative to the workspace root.